coss/opensource_clubs/templates/opensource_clubs/resources_page.html

```diff
@@ -0,0 +1,84 @@
+{% extends "home/about_page.html" %}
+{% load static wagtailcore_tags wagtailimages_tags gravatar %}
+
+{% block extra_css %}
+  <link rel="stylesheet" type="text/scss" href="{% static 'css/clubs.scss' %}">
+  <link rel="stylesheet" type="text/scss" href="{% static 'css/about.scss' %}">
+  <link rel="stylesheet" type="text/scss" href="{% static 'css/resources.scss' %}">
+{% endblock %}
+
+{% block brand-logo %}
+  <img src="{% static 'img/clubs-white.svg' %}" alt="mozilla">
+{% endblock %}
+
+{% block jumbotron %}
+{% endblock %}
+
+{% block content %}
+  <div class="content">
+    {% if page.heading_text %}
+      {% if page.heading_image %}
```

Comment:
Let's move the two `if`s in the same line with an `and` operator.

Reply:
By doing that we'll miss rendering that block in cases where we have text but not an image.